main.go

```diff
@@ -50,6 +51,10 @@ func main() {
 	}
 
 	cfgURL := "http://" + *cfgAddr + "/config/nsqlookupd_tcp_addresses"
+	if *httpAddrCfg {
+		cfgURL = "http://" + *cfgAddr + "/config/nsqlookupd_http_addresses"
+	}
```

Comment:
looks good. since the looks `httpAddrCfg` is used more like a boolean flag, was wondering if allowing the path to be passed would make little clearer?

```go
cfgPath = flag.String("config-path", "config/nsqlookupd_tcp_addresses", "The path of config endpoint")

cfgURL := fmt.Sprintf("http://%s/%s", *cfgAddr, *cfgPath)
```

Let me know your thoughts. I'm happy to merge it either way

Reply:
interesting approach, though different apps might accept different methods than the current (which is similiar)